timesketch/frontend-ng/src/components/Studio/SigmaRuleModification.vue

```diff
@@ -0,0 +1,257 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<template>
+  <v-container fluid>
+    <v-container fluid style="margin-bottom: 5px; margin-top: -79px">
+      <!-- This is a very hacky way to move the whole content to the top,
+      there is currently no global solution to get that depending on the context
+      TODO(jaegeral): find a better solution with jbn
+      -->
+      <strong>
+        {{ editingRule.title }}
+        <v-chip rounded x-small class="ml-2" :color="statusColors()">
+          <v-icon v-if="isParsingSuccesful" x-small> mdi-check </v-icon>
+          <v-icon v-else x-small> mdi-alert </v-icon>
+          {{ status_chip_text }}</v-chip
+        >
+      </strong>
+    </v-container>
+    <v-container fluid style="margin: 0px; padding: 0px; width: 50%">
+      <v-autocomplete
+        align="left"
+        dense
+        outlined
+        :items="SigmaTemplates"
+        @change="selectTemplate"
+        item-text="title"
+        label="Choose template"
+      >
+      </v-autocomplete>
+    </v-container>
+    <div class="alertbox" v-if="!isParsingSuccesful">
+      <v-alert dense type="error" outlined>
+        {{ status_text }}
+      </v-alert>
+    </div>
+
+    <v-textarea
+      label="Edit Sigma rule"
+      outlined
+      :color="statusColors()"
+      autocomplete="email"
+      rows="20"
+      v-model="ruleYamlTextArea"
+      background-color="statusColors()"
+      @input="parseSigma(ruleYamlTextArea)"
+      class="editSigmaRule"
+    >
+    </v-textarea>
+
+    <div>
+      <v-btn :disabled="!isParsingSuccesful" @click="addOrUpdateRule(ruleYamlTextArea)" small depressed color="primary">
+        {{ isNewRule ? 'Create Rule' : 'Update Rule' }}
+      </v-btn>
+      <div style="width: 10px; display: inline-block"></div>
+      <v-btn color="primary" text @click="$router.back()"> Cancel </v-btn>
+      <div style="width: 10px; display: inline-block"></div>
+      <v-btn @click="deleteRule(rule_uuid)" small text color="primary" :disabled="isNewRule"
+        ><v-icon>mdi-delete</v-icon></v-btn
+      >
+    </div>
+
+    <div class="alertbox" v-if="isParsingSuccesful">
+      <h3>Preview opensearch query:</h3>
+      <pre>{{ editingRule.search_query }}</pre>
+      <!-- TODO: This should become a global class how to nicely highlight a query -->
+    </div>
+  </v-container>
+</template>
+  
+<script>
+import ApiClient from '../../utils/RestApiClient'
+import { SigmaTemplates, defaultSigmaPlaceholder } from '@/utils/SigmaRuleTemplates'
+import EventBus from '../../main'
+import _ from 'lodash'
+
+export default {
+  props: ['rule_uuid', 'sigmaRule'],
+  data() {
+    return {
+      editingRule: { ruleYaml: defaultSigmaPlaceholder }, // empty state
+      status_chip_text: 'OK',
```

Comment:
Please make the case consistent (It's `OK` here and `Ok` in other places)

Reply:
done by just not using it anymore